bootstrap/README.md

```diff
@@ -31,7 +31,15 @@ terraform_state_bucket: <bucket>
 terraform_state_region: <region>
 ```
 
-Currently, this supports AWS, but we intend on supporting additional providers in the
+For example, for GCP:
+
+```yaml
+cloud_provider: gcp
+terraform_project_id: <project_id>
```

Comment:
This isn't in the `config.yaml.example` below. Add it and any other new configuration options that can be set.